artifactory/cli.go

```diff
@@ -854,15 +873,20 @@ func getDockerFlags() []cli.Flag {
 	return flags
 }
 
-func getNpmCommonFlags() []cli.Flag {
+//These flags are not valid for native npm command
```

Comment:
//These flags... --> // This flag...
...native npm command --> ...native npm commands.